src/main/java/com/GeorgesServer/app/Router.java

```diff
@@ -9,8 +9,15 @@
 
 public class Router {
 
+    private RequestLogger requestLogger;
+
+    public Router() {
+        this.requestLogger = new RequestLogger();
```

Comment:
it'd be cool to use the logger in our parser as well. Basically anywhere  where we might want to understand what happened in the event of an error.